doc/sphinx/refman-preamble.sty

```diff
@@ -16,10 +14,9 @@
 \newcommand{\forest}{\textsf{forest}}
 \newcommand{\Functor}{\kw{Functor}}
 \newcommand{\In}{\kw{in}}
-\newcommand{\Ind}[4]{\kw{Ind}[#2](#3:=#4)}
 \newcommand{\ind}[3]{\kw{Ind}~[#1]\left(#2\mathrm{~:=~}#3\right)}
```

Comment:
```suggestion
\newcommand{\Ind}[3]{\kw{Ind}~[#1]\left(#2\mathrm{~:=~}#3\right)}
```

Perhaps "ind" should be capitalized for consistency?

Reply:
It is possible but \ind is used in cic.rst and inductive.rst.
So, capitalizing \ind needs more changes.